### PR TITLE
IDE-856 Combine ECL files to a MOD file using right click menu in the repository

### PR DESCRIPTION
**Pull request by dehilsterlexis**
… repository

Signed-off-by: dehilsterlexis <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: wlib/RepositoryView.h [hunk 1]
```diff
@@ -453,6 +454,27 @@ class CRepositoryViewT
...
+				for (IAttributeVector::iterator itr = attrs.begin(); itr != attrs.end(); ++itr)
+				{
+					file.Write(IMPORT_MARKER);
+					file.Write(itr->get()->GetQualifiedLabel(true));
+					file.Write(_T("\n"));
+					file.Write(itr->get()->GetText());
+				}
+				file.Close();
+			}
+        }
```
Comment (GordonSmith) [inline wlib/RepositoryView.h]: Minor:  Indentation looks wrong.

Comment (dehilsterlexis) [reply wlib/RepositoryView.h]: Will fix.

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Comment by richardkchapman** [comment]
Should target 7.2.x

**Comment by dehilsterlexis** [comment]
@richardkchapman rebased

**Pull request merged**